FirebaseInstallations/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v1.1.2 -- M69
```

Comment:
Should have been a minor version update since dependencies are changing. Probably not worth fixing this late though.